docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,203 @@
+# Deploy Nebula Graph with Docker Compose
+
+There are multiple ways to deploy Nebula Graph, but using Docker Compose is usually considered to be the most convenient one.
+
+## Reading guide
+
+If you are reading this topic with the questions listed below, click them to jump to their answers.
+
+* [What do I need to do before deploying Nebula Graph?](#prerequisites)
+* [How to fast deploy Nebula Graph with Docker Compose?](#how-to-deploy)
+* [How to check the status and ports of the Nebula Graph service?](#check-the-nebula-graph-service-status-and-ports)
+* [How to check the data and logs of the Nebula Graph service?](#check-the-service-data-and-logs)
+* [How to stop the Nebula Graph Services?](#stop-the-nebula-graph-services)
+* [What are the other ways to install Nebula Graph?](#other-ways-to-install-nebula-graph)
+
+## Prerequisites
+
+* You have installed [Docker](https://docs.docker.com/engine/install/), [Docker Compose](https://docs.docker.com/compose/install/), and [Git](https://git-scm.com/download/linux) on your host.
+
+    >**NOTE**:
+    >
+    >* To use Docker as a non-root user, follow the steps in [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
+    >* We recommend that you install the latest version of these applications to make sure they work properly.
+
+* You have started the Docker service on your host.
+
+* If you have already deployed another version of Nebula Graph with Docker Compose on your host, to avoid compatibility issues，back up [the service data](#check-the-service-data-and-logs) if you need, and delete the `nebula-docker-compose/data` directory.
+
+## How to deploy
+
+1. Clone the `nebula-docker-compose` repository to your host with Git.
+
+    * To install Nebula Graph 1.0, clone the `master` branch.
+
+    ```shell
+    $ git clone https://github.com/vesoft-inc/nebula-docker-compose.git
+    ```
+
```

Comment:
This quick start is for Nebula Graph 2.0. Is it necessary to include this information?

Reply:
I'm worried about users' searching results. But yeah, we can keep it as clean as possible for now.